client/src/charts/wrapped_nivo/nivo_bar.js

```diff
@@ -32,10 +32,19 @@ const bar_table = (
           header: key,
           formatter: (value) =>
             _.isUndefined(value) ? "" : table_view_format(value),
+          text_align: (value) => {
+            const onlyAlphanum = value.replace(/[^W]+/g, "")
```

Comment:
Small regex error, that `W` is just the uppercase letter W, e.g. this is matching any character that isn't a "W". For the  "word" metacharacter, you need an escaped w, e.g. `/[^\W]+/g`.

As for the implementation, not all table values contain non-alphanumeric characters (e.g. bar graphs for people data; no $'s in people data, and only values larger than 999 contain a non-alphanumeric comma). Not all labels are free of non-alphanumeric characters either (e.g. some department names contain commas).

In an even bigger headache I just thought to check, the word metacharacter in regex doesn't match accented characters, so this would also break down for French text (oof, that's an annoying gotcha, gonna have to remember to run french text through `_.deburr` to strip accents before regex parsing it)!